### ISSUE TITLE
[TalkBack] Voice input is not labeled.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#253]
```
What steps will reproduce the problem?
Precondition:talkback On
1. Keypad button to select the audio input
2. Touch the keyboard icon that appears on the left side of the microphone icon
3. Image.50,Unlabelled


What is the expected output? What do you see instead?
Voice input labeled

What version of the product are you using? On what operating system?
Test Device : Nexus 4 (Please share the issue reproduction steps on Nexus 4)
Android OS version : talkback3.3.2,GoogleSearch 2.0.0.485486 & 2.4.10.626027
Test Application(Version):  (Please double check the issue using the latest 
beta version on eyes-free pjt)


Please provide any additional information below.

```

Original issue reported on code.google.com by `ezc...@gmail.com` on 29 Apr 2013 at 4:50

Attachments:
- [Voiceinput.png](https://storage.googleapis.com/google-code-attachments/eyes-free/issue-253/comment-0/Voiceinput.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 30 Apr 2013 at 7:42
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This is an issue with the Voice IME, and cannot be fixed from TalkBack. A bug 
has been filed against the Voice IME.
```

Original comment by `alanv@google.com` on 30 Apr 2013 at 7:43
- Changed state: **WontFix**
